src/w3c/headers.js

```diff
@@ -651,14 +653,27 @@ export function run(conf, doc, cb) {
       return urlParts[pos] || "";
     },
   });
-  if (conf.processVersion == "2014" || conf.processVersion == "2015" || conf.processVersion == "2017") {
-    pub(
-      "warn",
-      "Process " + conf.processVersion + " has been superceded by Process 2018."
-    );
-    conf.processVersion = "2018";
+  if (W3CProcessVersion.known.includes(conf.processVersion) === false) {
+    const msg =
+      `Unknown W3C Process Document version: "${
+        conf.processVersion
+      }". Please update your ` +
```

Comment:
Can this be in a single line: `"${conf.processVersion}"`?

Reply:
Prettier.js keeps formatting it like that... "computer says no" 😂

Reply:
Line 662 is okay but line 660 is not okay? That's weird 🤣

Reply:
I'm still hopeful we can delete all things (best kind of code). I have good feeling @plehegar will say "yes".